docs/self-hosting/advanced/sso-providers/auth0.mdx

```diff
@@ -77,8 +77,8 @@ When deploying LobeChat, you need to configure the following environment variabl
 | Environment Variable | Type | Description |
 | --- | --- | --- |
 | `ENABLE_OAUTH_SSO` | Required | Enable single sign-on (SSO) for LobeChat. Set to `1` to enable single sign-on. |
```

Comment:
Hey, `ENABLE_OAUTH_SSO` has been deprecate, just delete it. And did you think is a good idea to delete `SSO_PROVIDERS ` and `NEXTAUTH_SECRET ` in the sso provider pages and only keep this variables on the [auth page](https://lobehub.com/docs/self-hosting/advanced/authentication)?

Reply:
> Do you think is a good idea to delete `SSO_PROVIDERS` and `NEXTAUTH_SECRET` in the sso provider pages and only keep this variables on the [auth page](https://lobehub.com/docs/self-hosting/advanced/authentication)?

Is your suggestion that users should read the general auth page for the "core" set of environment variables, and then they should read their SSO-specific page for any SSO-specific environment variables? That does sound like a better way to keep things organized and up to date. But maybe that would be better as a separate PR, one that looks specifically for all the different ways these SSO-specific pages overlap with the general auth page?

Reply:
Yes, your understanding is very accurate, looking forward to your next PR.